### DISCUSSION TITLE
Changing cursor style to "not-allowed" when isDragReject is true is getting overridden.

### DISCUSSION THREAD

**Discussion by tim-ninan** [react-dropzone/react-dropzone#1305] [General]
Hello, I am trying to change the cursor type to be `not-allowed` over the drop zone area when the value of `isDragReject` is true. However, I notice that this style I am implementing is getting overridden during this action and the cursor has a green circle with a plus sign (the copy cursor icon). 

I looked over the Issues in `react-dropzone` and I came across this https://github.com/react-dropzone/react-dropzone/issues/1042. 

If I am understanding correctly, does this mean that I won't be able to implement the desired behavior I am looking for or is there another workaround someone is aware of? 

Thanks in advance for anyone's input about this.

**Comment by tim-ninan** [comment]
@rolandjitsu

**Comment by rolandjitsu** [comment]
@tim-ninan using `getRootProps` to apply a `style` on the element should be possible. Unless you're referring to the input, in which case you can use `getInputProps`. Checkout the https://react-dropzone.js.org/#basic-example.
